PCGen-base/gradle/reporting.gradle

```diff
@@ -33,4 +33,8 @@ jdepend {
 	sourceSets = []
 }
 
+task wrapper(type: Wrapper) {
+	gradleVersion = '4.4.1'
```

Comment:
this doesn't seem right. We already have this stanza in build.gradle.

Reply:
@grimreaper You mean in the build.gradle for this project or the main pcgen (pcgendev) project?

Reply:
ah, sorry, missed which project this was in. Lets keep them in the same place.

Reply:
NP:  So shift this to build.gradle and reporting.gradle will inherit it so to speak?

Reply:
yesplz

Reply:
new PR submitted.